Scheduler/v4/auxtel_fbs_spec_202209A.yaml

```diff
@@ -38,7 +38,7 @@ driver_configuration:
     script_configuration_spec:
       do_acquire: True
       acq_filter: empty_1
-      acq_grating: ronchi170lpmm
+      acq_grating: holo4_003
```

Comment:
This needs to be ronchi170lpmm

Reply:
Can we do the acquisition with holo4_003 or does it need to be ronchi170lppm? In the request they asked for the sequence 2xholo + 2xronchi so I figured it would be quicker to acquire with holo. I just update the _hot start configuration to match as I had left that one to acquire with ronchi.

Reply:
It has to be ronchi... but you can change the order of the observations to take ronchi first than holo...

Reply:
Got it, updated both to do the acquisition with ronchi